staycation-ts/src/components/accomodation-screen/accommodation-details/AccommodationDetails.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import './AccommodationDetails.css';
+
+const AccommodationDetails: React.FC<{ details: {
+        title: string,
+        subtitle: string,
+        description: string,
+        type: string
+        categorization: number,
+        personCount: number,
+        imageUrl: string,
+        freeCancelation: boolean,
+        price: number,
+        location: string,
+        postalCode: string
+    } }> = (props) => {
+    const startImage = require('../../../assets/images/star.png');
+    const calendarImage = require('../../../assets/images/calendar.png');
+
+    const categorizationRender = (categorization: number) => {
+        let content = [];
+
+        for(let i = 0; i < categorization; i++) {
+            content.push(<img src={ startImage } />);
+        }
```

Comment:
Slično kao i prije, ovdje također stavljaš slike zvjezdica u array, pa također doday **key** da izbjegneš error u consoli. Nadalje, budući da ovu funkciju već koristiš istu na dva mjesta, možeš dodati negdje jedan file naziva primjerice `helpers.tsx` i dodati tu funkciju u njega (uz `export` ključnu riječ na početku, tako da bi ju mogao importati na drugim mjestima gdje ti bude potrebna). Kasnije u taj file možeš dodavati i nove funkcije koje ćeš koristiti na više mjesta. To ti je radi preglednosti puno bolja opcija.